### DISCUSSION TITLE
i cant sync my work between vscode and github

### DISCUSSION THREAD

**Discussion by Ali-is-cool** [community/community#135245] [Codespaces] [answered]
### Select Topic Area

Question

### Body

when i try to sync my code in vscode to github an error message appears
![Screenshot (711)](https://github.com/user-attachments/assets/f7c54f9e-325d-46b8-9e5a-39e76114e31b)

**Comment by IsshikiHugh** [comment]
Could you show the log and the command output as well?

**Reply by Ali-is-cool** [reply]
![Screenshot (713)](https://github.com/user-attachments/assets/1b1d4569-64b6-4520-a9f7-01784752d5c5)
![Screenshot (712)](https://github.com/user-attachments/assets/585b1e49-7685-4a45-bd23-e0b25f43b68b)

**Reply by IsshikiHugh** [reply, marked as answer]
So the first screenshot tells the reason.
Just follow the instruction in the first screenshot. If you don't know how to run the command, just press `ctrl` `shift` `~` to open the built-in terminal of VSCode, and run `git config pull.rebase false` (for example) in your repository folder.

**Reply by Ali-is-cool** [reply]
Thank you so much!!!
